sdk/identity/identity/src/credentials/managedIdentityCredential/models.ts

```diff
@@ -2,12 +2,18 @@
 // Licensed under the MIT license.
 
 import { AccessToken, GetTokenOptions } from "@azure/core-auth";
+
 import { IdentityClient } from "../../client/identityClient";
 
 /**
  * @internal
  */
 export interface MSIConfiguration {
+  retryConfig: {
+    maxRetries: number;
+    startDelayInMs: number;
+    intervalIncrement: number;
```

Comment:
It's not clear to me what `intervalIncrement` does and why it's configurable. Can you provide more details. I don't understand why it's set to 2s.

Reply:
interval Increment is not 2 seconds it's just 2 - which means it's going to be doubled everytime.